src/main/java/com/rtb/projectmanagementtool/user/UserServlet.java

```diff
@@ -46,18 +49,20 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
This servlet handles "/user" requests. What exactly do we POST here?  I see add users operation below, but I believe there's also going to be an "update user" or maybe even a  "delete user" operation here.  We should think about how we would structure our servlets based on the different operations.  Either having multiple servlets (one per operation), or having a single servlet that takes an "operation" param.  I would suggest discussing with the team to come up with a common convention.